en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -4,38 +4,57 @@ Let's try organization login using a sample scenario.
 
 ## Scenario
 
-Pickup is an ambulance service company that has many employees who use different credentials to sign in to different internal enterprise applications. While Robert is an administrator at Pickup and the MedLife Hospital is the root organization while following are the sub-organizations and the respected admins.
```

Comment:
shall we stick to `root organization` instead of super organization?
@Sachin-Mamoru

Reply:
We have to separately address that issue, if we are addressing the root of the organization if it has sub-organizations, then that's fine. But if we are addressing the carbon super one then we might need to address it as a super organization. But when it goes to the deployment level I don't think we'll be having the super organization, instead, there will be an organization name there.
Since in the console it has super as the organization name
I have used that, otherwise might be confusing for a newcomer I guess